### PR TITLE
matching 'require' to repo URL

### PR DESCRIPTION
**Pull request by cliffordp**

### CODE REVIEW

File: doc/05-repositories.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ Example assuming you patched monolog to fix a bug in the `bugfix` branch:
     "repositories": [
         {
             "type": "vcs",
-            "url": "https://github.com/igorw/monolog"
+            "url": "https://github.com/monolog/monolog"
```
Comment (Camwyn) [inline doc/05-repositories.md]: This url is dead,
did you mean https://github.com/Seldaek/monolog ?

**Comment by xuanquynh** [comment]
This is just an example of documentation. Nothing need to be changed.

**Comment by cliffordp** [comment]
> This is just an example of documentation. Nothing need to be changed.

@xuanquynh the documentation is incorrect, as it does not match, regardless of whether it's `monolog/monolog` or `foo/bar`. I was simply matching this line to the `"require"` one below it.

**Comment by naderman** [comment]
You're deleting the very point of this documentation example: You can use a repository which does not match the package name and which is not the original package's repository. By applying this patch you would actually make it so the example no longer reflects what the section is trying to explain.

**Comment by cliffordp** [comment]
Ah, I'll have to take another look. Thanks for responding.

**Pull request closed without merging**